lib/organisms/components/banners/banner8/index.tsx

```diff
@@ -60,7 +66,7 @@ const BannerSection = ({ model }) => {
             prevButton={<span className="prev_arrow" />}
           >
             <Fragment>
-              {slides.map(slide => (
+              {slides.map((slide) => (
```

Comment:
```suggestion
             {model.children.map((slide, index) => {
             const slideData = convertDataModelToDataObject(bannerSlides);
```

I think we can write it like before without the need to construct the slides array from the begining

Reply:
done